packages/bbui/src/Form/Core/Checkbox.svelte

```diff
@@ -1,22 +1,23 @@
-<script>
+<script lang="ts">
   import "@spectrum-css/checkbox/dist/index-vars.css"
   import "@spectrum-css/fieldgroup/dist/index-vars.css"
   import { createEventDispatcher } from "svelte"
+  import { ChangeEventHandler } from "svelte/elements"
 
   export let value = false
-  export let id = null
-  export let text = null
+  export let id: string | undefined = undefined
+  export let text: string | undefined = undefined
   export let disabled = false
   export let readonly = false
-  export let size
+  export let size: string = "M"
   export let indeterminate = false
 
   const dispatch = createEventDispatcher()
-  const onChange = event => {
-    dispatch("change", event.target.checked)
+  const onChange: ChangeEventHandler<HTMLInputElement> = event => {
+    dispatch("change", event.currentTarget.checked)
```

Comment:
`currentTarget` can differ from `target` depending on how listeners are set up and if the event is bubbling. In this case they should be the same as the element that triggers the event (`target`) is the same as the element which the event listener is attached to (`currentTarget`), as they should both be the input element. But just something to keep in mind in future as this could be a breaking change if we were manually adding the listener at a higher level (like on the full document).

Reply:
`.target` doesn't exist on the event type, which is the only reason I changed it.

Reply:
This is sort of what I was alluding to in my comment below about the event types being difficult to work with - `target` 100% does exist in the real event, so I'm not sure why it doesn't in the type.